app/javascript/src/ChangePassword/index.jsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import React, {useState} from 'react'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import { LoginPage, Form, ActionGroup, Button } from '@patternfly/react-core'
+import { HiddenInputs, PasswordField, validateSingleField, FlashMessages } from 'LoginPage'
+import type {FlashMessage} from 'Types'
+
+import 'LoginPage/assets/styles/loginPage.scss'
+import brandImg from 'LoginPage/assets/images/3scale_Logo_Reverse.png'
+import PF4DownstreamBG from 'LoginPage/assets/images/PF4DownstreamBG.svg'
+
+type Props = {
+  lostPasswordToken: ?string,
+  url: string,
+  errors: (?FlashMessage)[]
+}
+
+const renderInputWithProps = (name, label, value, isValid, setValue, setIsValid, validateSingleField) => {
+  const props = {
+    isRequired: true,
+    name: `user[${name}]`,
+    fieldId: `user_${name}`,
+    label,
+    value,
+    isValid,
+    onChange: (value, event) => {
+      const isValid = validateSingleField(event)
+      setValue(value)
+      setIsValid(isValid)
+    }
+  }
+  return <PasswordField inputProps={props} />
+}
+
+const ChangePassword = ({lostPasswordToken, url, errors}: Props) => {
```

Comment:
Let's use index only to export components and functions and create a `ChangePassword.jsx` apart.